examples/expressions/10_expressions_intro.py

```diff
@@ -2,49 +2,58 @@
 .. _example_expressions_intro:
 
 
-Building complex tabular pipelines
-==================================
+Building a predictive model by combining multiple tables with the skrub Data Plan
+====================================================================
+This example summarizes the use of skrub DataOps to build a ``Learner``:  recording
+all the Data Operations involved in building a predictor starting from one or
+more tables, and then replays them on unseen data.
+
+Here, we consider the problem of predicting whether the e-commerce transactions
+in a dataset are fraudulent: to do so, we need to train a classifier that takes
+a decision based on the features of each transaction, with the additional complication
+that the features are not stored in the table that contains the labels.
 
-In this example, we show a simple pipeline handling a dataset with 2 tables,
-which would be difficult to implement, validate, and deploy correctly without
-skrub.
 """
 
 # %%
 # The credit fraud dataset
 # ------------------------
 #
-# This dataset comes from an e-commerce website. We have a set of "baskets" (
-# orders that have been placed with the website). The task is to detect which
-# orders were fraudulent (the customer never made the payment).
-#
+# This dataset comes from an e-commerce website. The information is found in two
+# separate tables: the "baskets" table contains the labeled orders (for which
+# we know whether they are legitimate or not); the "products" table, which
+# contains the content of all baskets, including those for which we do not have
+# a label. These "unlabeled" baskets represent our "unseen data", which is what
+# the predictor would encounter in practice after being put in production.
 # The ``baskets`` table contains a basket ID and a flag indicating if the order
 # was fraudulent or not.
-
+# We start by loading the ``baskets`` table, and exploring it with the ``TableReport``.
+#
 # %%
 import skrub
 import skrub.datasets
 
-dataset = skrub.datasets.fetch_credit_fraud()
+dataset = skrub.datasets.fetch_credit_fraud()  # load labeled data
 skrub.TableReport(dataset.baskets)
```

Comment:
Nitpick: could we import the TableReport directly to replace `skrub.TableReport` with `TableReport`?